frontend/app/last-comments.tsx

```diff
@@ -55,9 +57,15 @@ async function init(): Promise<void> {
       (node.dataset.max && parseInt(node.dataset.max, 10)) ||
       remark_config.max_last_comments ||
       DEFAULT_LAST_COMMENTS_MAX;
-    getLastComments(remark_config.site_id!, max).then(comments => {
+    const locale = remark_config.locale || 'en';
```

Comment:
I meet line like this second time. Maybe it should be placed in `StaticStore`?

Reply:
We have very same last-comments.tsx and remark.tsx. It is not problem this request.

Reply:
But I move getLocale to separate module.